tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,1650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.                        data = dask_cudf.read_parquet(parquet_path,storage_options={'anon': True, 'use_ssl': False}).sample(frac=data_frac, random_state=seed)
</pre><p>Remove extra arguments being used here as well as remove data_fra. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I think <code>data_frac</code>  if fine as is. For completeness one may try to run this somewhere else with a smaller chunk of dataset. However I should remove the <code>model_args={}, model_kwargs={}</code>  though. Good Catch.</p>